src/remat.jl

```diff
@@ -522,11 +522,17 @@ function σs(A::ReMat{T,1}, sc::T) where {T}
     NamedTuple{(Symbol(only(A.cnames)),)}(sc*abs(only(A.λ.data)),)
 end
 
-function σs(A::ReMat{T}, sc::T) where {T}
-    λ = A.λ.data
-    NamedTuple{(Symbol.(A.cnames)...,)}(ntuple(i -> sc*norm(view(λ,i,1:i)), size(λ, 1)))
+function _σs(λ::LowerTriangular{T}, sc::T, cnames) where {T}
```

Comment:
It's too bad there isn't a non allocating / view-based version of `LinearAlgebra.diag`

Reply:
I'm not sure I understand the context of this comment.  In the LowerTriangular case you need access to all the nonzeros in λ to be able to calculate the standard deviation of the random effects on the original scale

Reply:
A thinko on my part! I was thinking about just the diagonal and not all the nonzeros.